CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+## Contributing and Developer guide
+
+The steps outlined below are based on the assumption that you are building the ROCK with the Ubuntu Jammy (22.04).  If you are using another version of Ubuntu or another operating system, the process may be different.
+
+### Clone Repository
+```bash
+git clone git@github.com:canonical/charmed-spark-rock.git
+cd charmed-spark-rock
+```
+
+### Installing Prerequisites
+```bash
+sudo snap install rockcraft --edge
+sudo snap install docker
+sudo snap install lxd
+sudo snap install skopeo --edge --devmode
+```
+
+### Configuring Prerequisites
+```bash
+sudo usermod -aG docker $USER 
+sudo lxd init --auto
+```
+*_NOTE:_* You will need to open a new shell for the group change to take effect (i.e. `su - $USER`)
```

Comment:
it should be sufficient to run the following command instead of open a new shell: 
```
newgrp docker
sudo snap disable docker
sudo snap enable docker
```